src/core/tests/type_prop/bitwise_ops.hpp

```diff
@@ -0,0 +1,1020 @@
+    const auto op = std::make_shared<TypeParam>(param_A, param_B, op::AutoBroadcastType::NONE);
+
+    auto out_shape = op->get_output_partial_shape(0);
+
+    EXPECT_EQ(out_shape, expected_shape);
+    EXPECT_EQ(get_shape_labels(out_shape), get_shape_labels(expected_shape));
+}
+
+TYPED_TEST_P(BitwiseOperator, labels_equal_static_shape_broadcast_none) {
+    // Static shape
+    ov::PartialShape pshape_A{2, 3, 224, 128};
+    ov::PartialShape pshape_B{2, 3, 224, 128};
+    ov::PartialShape expected_shape{2, 3, 224, 128};
+
+    // Equal labels
+    set_shape_labels(pshape_A, {30, 31, 32, 33});
+    set_shape_labels(pshape_B, {30, 31, 32, 33});
+    set_shape_labels(expected_shape, {30, 31, 32, 33});
+
+    auto param_A = std::make_shared<ov::op::v0::Parameter>(element::i32, pshape_A);
+    auto param_B = std::make_shared<ov::op::v0::Parameter>(element::i32, pshape_B);
+    const auto op = std::make_shared<TypeParam>(param_A, param_B, op::AutoBroadcastType::NONE);
+
+    auto out_shape = op->get_output_partial_shape(0);
+
+    EXPECT_EQ(out_shape, expected_shape);
+    EXPECT_EQ(get_shape_labels(out_shape), get_shape_labels(expected_shape));
+}
+
+TYPED_TEST_P(BitwiseOperator, labels_different_dynamic_shape_broadcast_none) {
+    // Dynamic shape
+    ov::PartialShape pshape_A{ov::Dimension(-1), ov::Dimension(3), ov::Dimension(2, 224), ov::Dimension(1, 128)};
+    ov::PartialShape pshape_B{ov::Dimension(-1), ov::Dimension(3), ov::Dimension(2, 224), ov::Dimension(1, 128)};
+    ov::PartialShape expected_shape{-1, 3, ov::Dimension(2, 224), ov::Dimension(1, 128)};
+
+    // Different labels
+    set_shape_labels(pshape_A, {10, 11, 12, 13});
+    set_shape_labels(pshape_B, {20, 21, 22, 23});
+    set_shape_labels(expected_shape, {20, 21, 22, 23});
+
+    auto param_A = std::make_shared<ov::op::v0::Parameter>(element::i32, pshape_A);
+    auto param_B = std::make_shared<ov::op::v0::Parameter>(element::i32, pshape_B);
+    const auto op = std::make_shared<TypeParam>(param_A, param_B, op::AutoBroadcastType::NONE);
+
+    const auto out_shape = op->get_output_partial_shape(0);
+
+    EXPECT_EQ(out_shape, expected_shape);
+    EXPECT_EQ(get_shape_labels(out_shape), get_shape_labels(expected_shape));
+}
+
+TYPED_TEST_P(BitwiseOperator, labels_equal_dynamic_shape_broadcast_none) {
+    // Dynamic shape
+    ov::PartialShape pshape_A{ov::Dimension(-1), ov::Dimension(3), ov::Dimension(2, 224), ov::Dimension(1, 128)};
+    ov::PartialShape pshape_B{ov::Dimension(-1), ov::Dimension(3), ov::Dimension(2, 224), ov::Dimension(1, 128)};
+    ov::PartialShape expected_shape{-1, 3, ov::Dimension(2, 224), ov::Dimension(1, 128)};
+
+    // Equal labels
+    set_shape_labels(pshape_A, {30, 31, 32, 33});
+    set_shape_labels(pshape_B, {30, 31, 32, 33});
+    set_shape_labels(expected_shape, {30, 31, 32, 33});
+
+    auto param_A = std::make_shared<ov::op::v0::Parameter>(element::i32, pshape_A);
```

Comment:
```suggestion
    auto a = std::make_shared<ov::op::v0::Parameter>(element::i32, pshape_A);
```
In other test this is `A`, Select one naming convention.

Reply:
Fixed